package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "05-recipe-card-esnext",
```

Comment:
update the fields to the appropriate values.

Reply:
updated!